misc/formatter/modules/formatter-core/src/test/resources/declarations/class-definition/assert/class_definition_declaration_13.bal

```diff
@@ -1,5 +1,6 @@
 readonly distinct class Foo {
     string name;
+
```

Comment:
This also looks wrong. Lets keep the original

Reply:
Isn't it a explicitly preserved newline added by the user?